src/cli.rs

```diff
@@ -50,6 +50,12 @@ struct Cli {
     #[arg(long, global = true, required = false)]
     cert_pem: Option<String>,
 
+    /// This option is for passing a file path to a pem-encoded TLS certificate string to establish
+    /// a connection with the LNDK server. If this isn't set, the cli will look for the TLS file in
+    /// the defaultlocation (~.lndk).
```

Comment:
nit/typo: default location. Also if you wouldn't mind just adding a quick note at the end of the docs for both cert_path and cert_pem, something like: "Only one of cert_pem or cert_path can be set at once."